spec/fortnox/api/repositories/unit_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'fortnox/api'
+require 'fortnox/api/mappers'
+require 'fortnox/api/repositories/customer'
+require 'fortnox/api/repositories/examples/all'
+require 'fortnox/api/repositories/examples/find'
```

Comment:
This import is currently not used. I guess that's because there are no searchable attributes in this model. Since we have a find method on this repository even though there are no searchable attributes, we should probably test that the `find`-method behaves as expected (probably by throwing some error).